virtualbox/vbox-adapter-check.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+
+import sys
+import textwrap
+import argparse
+import virtualbox
+from virtualbox.library import NetworkAttachmentType as NetType
+from gi.repository import Notify
```

Comment:
What are the dependencies for this script? Where are they specified?

Typically this would go in a `requirements.txt` file, though with a bunch of disparate scripts, keeping them in the file header can also work. Recently, I've been migrating to using `uv run` to invoke single-file scripts with dependencies in the header: https://docs.astral.sh/uv/guides/scripts/#declaring-script-dependencies

Reply:
the only dependency that shouldn't be already installed in Debian with GNOME is `virtualbox`.

Reply:
also, it may be worth catching `ImportError`s and explaining to the user when things don't work:

  - for `virtualbox`: what they need to do to install the bindings
  - for `gi`: that the script must be run on a system with Gnome

Reply:
>it may be worth catching `ImportError`s and explaining to the user when things don't work:

I like this idea. I also like the idea of the `requirements.txt` file, but it is a bit tricky with the current structure as all the scripts are in the same folder an they could have different dependencies. :thinking: 

>   - for `virtualbox`: what they need to do to install the bindings

The `virtualbox` library is required by all the scripts in the `virtualbox` folder. so if we add this, we should add it to all the scripts for consistency

>   - for `gi`: that the script must be run on a system with Gnome

I think `gi` may work on other types of systems as well, but I am not sure in which. I have tried this notification in openSUSE with KDE and it seems to work.

Reply:
I have been looking into adding this and I think we should leave it out, the error looks like:
```
Traceback (most recent call last):
  File "/home/anamg/github/flare-vm/virtualbox/./vbox-adapter-check.py", line 7, in <module>
    import virtualbox
ModuleNotFoundError: No module named 'virtualbox'
```
You can install this library the same way as any other Python library (`python3 -m pip install virtualbox`), so I think a custom message does not help more than the generic error for which you can find help online. @williballenthin are you ok with merging it as it is? We can discuss how to improve if needed afterwards.